runtests.py

```diff
@@ -47,6 +47,7 @@
     __doc__ = __doc__.format(**globals())
 
 
+import builtins
```

Comment:
Why is it necessary to import `builtins` here?

Reply:
exec is a built-in function

Reply:
removed the import (it was there because it was currently being imported and I was trying to change as little as possible).